andela/suyabay
# Align password reset form to be properly placed

#### What does this PR do?

Align password reset form to be properly placed
#### Description of Task to be completed?
- Fix password reset form alignment
#### How should this be manually tested?
- When users visit `http://suyabay-staging.herokuapp.com/passwordreset` they should see a well aligned password reset form. 
#### Any background context you want to provide?
#### What are the relevant pivotal tracker stories?
#119263559
#### Screenshots (if appropriate)

![screen shot 2016-05-10 at 12 53 45 pm](https://cloud.githubusercontent.com/assets/14159435/15145648/c9643fb8-16ae-11e6-90b7-024196f0c42b.png)

![screen shot 2016-05-10 at 12 58 02 pm](https://cloud.githubusercontent.com/assets/14159435/15145674/de4be62e-16ae-11e6-87b5-bd4b3547083a.png)
#### Questions:

[@unicodeveloper](https://github.com/unicodeveloper)
[@andela-oadebayo](https://github.com/andela-oadebayo)
